octoprint_mrbeam/iobeam/lid_handler.py

```diff
@@ -553,7 +565,28 @@ def add_to_stat(pos, avg, std, mass):
 
 		except Exception as ex:
 			self._logger.exception('Exception_in-_save__s_for_analytics-_{}'.format(ex))
-
+	
+	def capture_last_img_for_analytics(self):
```

Comment:
Name is misleading: It sounds like you are taking an extra picture for the pupose of the analytics.

If you want to upload a picture after the fact ( the client already received the picture, then decides to send it to the analytics ), then it would be best to send the latest image at the start of the serve_pictures function (for example at line 425 of this file) and not during the preparation of the image.

Reply:
Or bypass the lidhandler completely by reading the cam/debug/undistorted.jpg file, encode and then send it.